.github/CODEOWNERS

```diff
@@ -7,6 +7,13 @@
 # cmake
 /cmake/ @hikinggrass @a-w50
 
+# lib
```

Comment:
Please always put at least two people for each specification

Reply:
done

Reply:
done